forte/data/data_store.py

```diff
@@ -1212,11 +1215,40 @@ def co_iterator_annotation_like(
 
         Args:
             type_names: a list of string type names
+            range_annotation: a list with two elements,
+                the start and the end of the range in
+                which we want to get required entries
 
         Returns:
             An iterator of entry elements.
         """
 
+        def get_bisect_range(search_list: SortedList):
+            """
+            Perform binary search on the specified list for target entry class.
+            Args:
+                entry_class: Name of the target type of entry. It can be Annotation or
```

Comment:
subtype of `Annotation` or `AudioAnnotation`.

Btw, to make this docstring actually useful, we should refer these classes with the `:class:` sphinx syntax

Reply:
Got it!